src/components/Form.js

```diff
@@ -120,20 +121,32 @@ function Form(props) {
 
     const hasServerError = useMemo(() => Boolean(props.formState) && !_.isEmpty(props.formState.errors), [props.formState]);
 
+    /**
+     * This function is used to remove invisible characters from strings before validation and submission.
+     *
+     * @param {Object} values - An object containing the value of each inputID, e.g. {inputID1: value1, inputID2: value2}
+     * @returns {Object} - An object containing the processed values of each inputID
+     */
+    const prepareValues = useCallback((values) => {
```

Comment:
`useMemo` is more accurate here.
Actually, we can take this function out of component lifecycle as it requires no dependency.